src/main/java/com/softserve/paymentservice/service/AmountCalculator.java

```diff
@@ -0,0 +1,33 @@
+package com.softserve.paymentservice.service;
+
+import com.softserve.paymentservice.dto.PaymentInfoDto;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AmountCalculator {
+
+    private int amount;
+
+    public enum Tariffs {
+        BASIC(10), PREMIUM(1200);
+        private int coefficient;
+
+        Tariffs(int coefficient) {
+            this.coefficient = coefficient;
+        }
+    }
+
+    public int calculateAmount(PaymentInfoDto paymentInfoDto) {
+        if (paymentInfoDto.getTariff().equalsIgnoreCase(Tariffs.PREMIUM.toString())) {  //Tariffs.valueOf(tariff).toString().equals("PREMIUM")
```

Comment:
prefer case-sensitive equals if you business logic doesn't require it to case-insensitive